### ISSUE TITLE
Make brew formula use Python3

### ISSUE THREAD

**Issue by tsroten** [dbcli/mycli#570]
The current brew formula works fine, but it would be nice to make it use brew's Python 3 instead of Python 2.

**Comment by engelju** [comment]
Yup, it would be nice to have this, so that you don't have to use a python2 dependency.

Link to the brew formulae: http://formulae.brew.sh/formula/mycli

**Comment by amjith** [comment]
This is no longer relevant. Python 2 is long gone.
